README.md

```diff
@@ -72,37 +73,37 @@ peer.registerReceiveHandlerForMessageType("testMessageType", async (message: Mes
 // Add custom receive handler with a response type
 peer.registerReceiveHandlerForMessageType("testMessageTypeWithResponse", async (message: Message, senderGuid: string, response: RESPONSE_TYPE) => {
     if (message && senderGuid) {
-        // Rspond to the message
+        // Respond to the message
         response(message);
     }
 });
 
 // Get a validator peer
-const validatorPeer: [string,string] = peer.getPeerOfType(PeerType.VALIDATOR); 
-if (!validatorPeer) {
+const validatorPeer: [string, string] = peer.getPeerOfType(PeerType.VALIDATOR); 
+if (validatorPeer) {
     // Get the guid of the validator. [0] = Guid, [1] = Ip 
     const validatorGuid: string = validatorPeer[0];
     
     // Basic message without responses
-    const message: Message = new Message("testMessageType", "testMessageType");
+    const messageBasic: Message = new Message("testMessageType", "testMessageType");
```

Comment:
The comment reads `// Basic message`, should the variable not be called the same?

Reply:
This was done to make a distinction between a message without a response (messageBasic) and a message with a response (messageWithResponse) while keeping a consitante naming scheme: message[typeOfMessage].